bitnami/README.md

```diff
@@ -45,3 +47,12 @@ pubsub
 * add more resources to be base if needed
 * PR the changes
 * use the new base in `kubernetes-manifests`
+
+## Monitoring
```

Comment:
Maybe split those links by cluster, as they are avaible in dev, prod accross all providers aws, merit and gcp

Reply:
> Maybe split those links by cluster, as they are avaible in dev, prod accross all providers aws, merit and gcp

[d0bd7ab](https://github.com/utilitywarehouse/kafka-manifests/pull/59/commits/d0bd7abbb03c70581b5d95a992e7b98e8a7be3cd)